src/Tribe/Views/V2/Widgets/Widget_QR_Code.php

```diff
@@ -235,10 +233,18 @@ public function setup_admin_fields() {
 			];
 
 			$series = get_posts( $args );
-			foreach ( $series as $series ) {
+			if ( ! empty( $series ) ) {
+				$series_options = [];
```

Comment:
This variable should be defined above line 221, before the start of the `if` block.